packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-ordering/.field-initializers-after-methods-private/output.js

```diff
@@ -0,0 +1,34 @@
+var _dec, _init_foo, _init_extra_foo;
+var counter = 0;
+_dec = (_, {
+  addInitializer
+}) => {
+  addInitializer(function () {
+    counter++;
+    expect(typeof this.method).toBe("function");
+    expect(this.#foo).toBe("#foo");
+    expect(() => this.#bar).toThrow();
+  });
+};
```

Comment:
This test is [`field-initializers-after-methods`](https://github.com/babel/babel/blob/main/packages/babel-plugin-proposal-decorators/test/fixtures/2023-05-ordering/field-initializers-after-methods/exec.js) with private flavour. Unfortunately our output is not valid because the decorator is hoisted out of the class. This test is disabled temporarily. This issue is related to https://github.com/babel/babel/issues/16176.